README.md

```diff
@@ -36,6 +36,60 @@ Incidents are typically categorized among following types:
 -   **Infrastructure Event**
     -   Used for integration for system management tools.
 
+The **Authentication Mechanism to Use** config parameter provides the configuration information Phantom uses to communicate with BMC Platform. Currently, there are two ways to authenticate.
```

Comment:
Can you change the `Phantom` reference to `Splunk SOAR`?

Reply:
Sure, I have replace phantom to Splunk SOAR.

Reply:
Hello @splunk-jessica, I did the suggested changes. Could you please review and resolve this thread.